public/app/features/explore/state/actions.ts

```diff
@@ -572,7 +587,7 @@ export function removeQueryRow(exploreId: ExploreId, index: number): ThunkResult
 /**
  * Main action to run queries and dispatches sub-actions based on which result viewers are active
  */
-export function runQueries(exploreId: ExploreId) {
+export function runQueries(exploreId: ExploreId, ignoreUIState = false) {
```

Comment:
not sure I understand this flag, when this relevant?

Reply:
When requesting queries on Explore init. Whether or not the queries are issued depends on the state of the panels (which sort of makes sense). But, when entering Explore for the first time, I'm ignoring this, and issuing quieries anyway -> that's why the problem you have described below occurs. 

The queries are issued based on the `show[Graph|Logs|Table]` properties in state (and that's implemented in `runQueries` action), while they execution could have been issued from the panels (on mount / toggle). Gonna think what's there to be refactored to achive that.